xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-macro/plugin.js

```diff
@@ -754,6 +751,8 @@
       }
       var updatingWidget = !!widget?.element;
       if (updatingWidget && widget.element.getName() === expectedElementName) {
+        // we remove all nested editable so that they're rebuilt.
```

Comment:
The nested editables are going to be rebuilt anyway because the content is going to be refreshed. The reason you remove them is probably because you want the server to receive only what is available in the macro call, otherwise the content from the nested editables is overwriting the values from the macro call.

Reply:
I forgot to improve that comment I'll do it.